sbin/get-redisjson

```diff
@@ -82,6 +82,13 @@ else
 		else
 			nick="rhel7"
 		fi
+	elif [[ $dist == mariner ]]; then
+		OSS=1
+		if [[ -z $REPO_PATH ]]; then
```

Comment:
I couldn't understand how `$REPO_PATH` determines is we're running on master on not...

Reply:
You are right, the comparison was modified to use the `BRANCH` variable